core/primitives/src/account.rs

```diff
@@ -24,6 +28,45 @@ impl Account {
     }
 }
 
+/// Calculates the storage and the name rent for the given account
+/// for period of account.storage_paid_at to current block_height
+pub fn calculate_rent(
+    account_id: &AccountId,
+    account: &Account,
+    // TODO #1903 block_height: BlockHeight,
+    block_index: BlockHeight,
```

Comment:
Please address todo

Reply:
I'll change it here locally, but I don't see if there is a consensus around the issue: https://github.com/nearprotocol/nearcore/issues/1903

Reply:
In nearcore we should use block_height, but when exposing the API in vm-logic we should use the old API name, even though internally it's using the new field.